cluster/saltbase/salt/fluentd-gcp/fluentd-gcp.yaml

```diff
@@ -9,6 +9,7 @@ spec:
     resources:
       limits:
         cpu: 100m
+        memory: 200Mi
```

Comment:
Isn't 100Mi more than enough given that in all of Saad's tests it never used that much?